sirepo/util.py

```diff
@@ -187,8 +187,10 @@ def flask_app():
     return flask.current_app or None
 
 def in_flask_request():
+    import sys
+    if 'flask' not in sys.modules:
+        return None
     import flask
```

Comment:
f = sys.modules.get('flask')
return f and f.request or None

Reply:
a3911a26